BASE/src/Microsoft.ApplicationInsights/Extensibility/Implementation/Tracing/SelfDiagnosticsInternals/SelfDiagnosticsConfigParser.cs

```diff
@@ -0,0 +1,128 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Implementation.Tracing.SelfDiagnosticsInternals
```

Comment:
There was a conflict with the class in my last PR. I've renamed that into `SelfDiagnosticsInitializer` as well so that the namespace has the `SelfDiagnostics` name.